src/vtbackend/Functions_documentation.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <vtbackend/Functions.h>
+
+#include <string_view>
+
+namespace terminal::documentation
+{
+
+// NOTE: This file currently serves more as an experiment, on how we could actually start documenting
+//       the various functions. It is not yet used anywhere.
+//
+//       The goal is to have it all constexpr, and that we can generate documentation from it
+//       for markdown, website, and man-pages.
+
+struct Documentation
+{
+    FunctionDefinition sequence;
```

Comment:
We want to include documentation in config entry as well, so not sure that `FunctionDefinition` can be inside here, maybe some base class and then some additional class for documentation of VT seqs with `FunctionDefinition`

Reply:
I should probably not call it `Documentation` in a too common namespace then. you are right.
But config entry definition and documentation should be done in the frontend code and, thus, not conflict with what is in vtbackend.